frontend/src/components/dashboard/create/govsg/GovsgMessages.tsx

```diff
@@ -85,6 +97,11 @@ export const GovsgMessages = ({ campaignId }: GovsgMessagesProps) => {
     void fetchGovsgMessages(index)
```

Comment:
Does this need to be changed? Consider the following scenario:
1. The user has performed a search for phone numbers starting with '9' and there are multiple pages returned
2. When the user navigates to the next page, this will cause the search params to be reset

We might need to ensure that all calls to `fetchGovsgMessages` contain the `search` variable.

Reply:
I think so. I have not been successful with bulk campaigns so I can't verify this behaviour.

My qn to the team about "un-stuck-ing" campaigns wasn't being replied to, got to ping them again.

Reply:
Reproduced locally.

Have done: `ensure that all calls to fetchGovsgMessages contain the search variable`.